server/container/guice/jmx/src/main/java/org/apache/james/modules/server/JMXServer.java

```diff
@@ -126,4 +146,38 @@ private void doStop() {
         }
     }
 
+    private void generateJMXPasswordFileIfNeed() {
+        if (Boolean.parseBoolean(System.getProperty(JMX_CREDENTIAL_GENERATION_ENABLE_PROPERTY, JMX_CREDENTIAL_GENERATION_ENABLE_DEFAULT))
+            && !existJmxPasswordFile()) {
+            generateJMXPasswordFile();
+        }
+    }
```

Comment:
Can we have the CLI attempt to read `~/conf/jmxremote.password` that is in clear if no option is supplied?